internal/sdkprovider/service/kafka/mirrormaker_replication_flow.go

```diff
@@ -102,6 +103,15 @@ var aivenMirrorMakerReplicationFlowSchema = map[string]*schema.Schema{
 		Description:  "Offset syncs topic location. Possible values are `source` & `target`. There is no default value.",
 		ValidateFunc: validation.StringInSlice([]string{"source", "target"}, false),
 	},
+	"config_properties_exclude": {
```

Comment:
Hm only issue i might see is that we have max 2048 chars string here in [API](https://github.com/aiven/aiven-core/blob/15e1e5933c0d568433d884e21baf6268d79835e4/aiven/web/schema.py#L3660). Could potentially be confusing if it fails due to this but i don't think a major issue.